### DISCUSSION TITLE
My github account cannot be searched in google

### DISCUSSION THREAD

**Discussion by marcoAu109** [community/community#43542] [New to GitHub]
my github marcoAu109 cannot be searched in google. i have studies some advices but not able to handle the verification of site ownership in google. Can any fellow help? Really appreciate!!!

**Comment by jfullstackdev** [comment]
it has something to do with web crawlers, the more it's being search the more it's being located, remembered and linked to other social medial, like me, I'm using same Twitter handle

but back then, I really wanted to make it fast, so I had Google Analytics for that and Google Search Console, then yes, it's now index, even my posts in Twitter

**Reply by marcoAu109** [reply]
Thank you. Can you explain more how to use Google search console and Google Analytics? thanks in advance

**Comment by melvincwng** [comment]
Hey @marcoAu109 👋🏻, for your GitHub account to be searchable on Google, you need to wait for Google's webcrawlers to crawl through your GitHub page, and then index it in their search engine. This is how it works behind the scenes. It usually take around [a few days to a few weeks](https://developers.google.com/search/docs/crawling-indexing/ask-google-to-recrawl#:~:text=Crawling%20can%20take%20anywhere%20from,instantly%20or%20even%20at%20all.) for Google to crawl through a fresh website/webpage, so please be patient during this process.

Nonetheless, when I last searched your GitHub username on Google, it seems like your GitHub profile page has already been crawled & indexed by Google. For reference, here is a screenshot:

![image](https://user-images.githubusercontent.com/77479885/211147964-9072de82-6231-4d18-9bdf-669acc4bd08f.png)

Hope this answers your question, take care & have a good day 😄

**Reply by marcoAu109** [reply]
Thank you so much Melvin
